pkg/monitors/forwarder/forwarder.go

```diff
@@ -83,8 +83,9 @@ func setupHandler(ctx context.Context, router *mux.Router, chainType string, sin
 }
 
 func notFoundHandler(w http.ResponseWriter, r *http.Request) {
-	errMsg := fmt.Sprintf("Datapoint or span request received on invalid path '%s'. "+
-		"You should send to the same path that you would on the Smart Gateway.", r.URL.Path)
-	logger.ThrottledError(errMsg)
-	_, _ = w.Write([]byte(errMsg + "\n"))
+	errMsg := "Datapoint or span request received on invalid path"
+	logger.ThrottledError(fmt.Sprintf("%s: %s", errMsg, r.URL.Path))
+	errMsg = fmt.Sprintf("%s. You should send to the same path that you would on the Smart Gateway.\n", errMsg)
```

Comment:
Actually this is an obsolete message since we don't really use Gateway anymore.  This ideally would just be a list of the supported paths for maximum clarity.

Reply:
Updated w/ supported paths.